schema/types/isolator.go

```diff
@@ -52,6 +77,24 @@ func (is *Isolators) GetByName(name ACIdentifier) *Isolator {
 	return nil
 }
 
+// ReplaceIsolatorsByName overrides matching isolator types with a new
+// isolator, deleting them all and appending the new one instead
+func (is *Isolators) ReplaceIsolatorsByName(newIs Isolator, oldNames []ACIdentifier) uint {
+	var numReplacements uint = 0
```

Comment:
Why bothering counting numReplacements? the only caller of this function ignore the returned value.